2108.02159/2108.02159.yaml

```diff
@@ -168,50 +178,53 @@
       score:
         Haining: 2
         Will:
-        Yasaman:
+        Yasaman: 1
+        # Would have preferred \Delta(r)
     real|momentum:
       LLM: real
       human: momentum
       score:
         Haining: 0
         Will:
-        Yasaman:
+        Yasaman: 2
+        # Both seem valid answers as both appear in excerpt
```

Comment:
Eq. (1) is in real space.
Eq. (3) is in k space. 
I wish it to give Eq. (3) directly, so that's way I put "momentum" here